casanovo/casanovo.py

```
@@ -555,61 +680,78 @@ def setup_model(
+            _DEFAULT_MODEL_ID,
+            _DEFAULT_MODEL_ID,
+        )
+        model = _DEFAULT_MODEL_ID
+        try:
+            resolved_model = _get_model_weights(model, cache_dir, version)
+        except github.GithubException.RateLimitExceededException:
+            logger.error(
+                "GitHub API rate limit exceeded. Download model weights "
+                "manually from https://github.com/Noble-Lab/casanovo "
+                "and use '--model <path>'."
             )
```

Ruff (0.15.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)